dgraph/cmd/alpha/run.go

```diff
@@ -614,17 +620,12 @@ func run() {
 	}
 
 	bindall = Alpha.Conf.GetBool("bindall")
-
-	totalCache := int64(Alpha.Conf.GetInt("cache_mb"))
+	cache := z.NewSuperFlag(Alpha.Conf.GetString("cache")).MergeAndCheckDefault(
+		worker.CacheDefaults)
+	totalCache := cache.GetInt64("size-mb")
 	x.AssertTruef(totalCache >= 0, "ERROR: Cache size must be non-negative")
-	if Alpha.Conf.IsSet("lru_mb") {
```

Comment:
@ajeetdsouza we are anyways breaking the flags with super flag changes. So this removal should be okay, I believe.

Reply:
Yup, this should be good. We were going to remove the `lru_mb` flag in the next release anyway.